Frontend/business-expense-manager/src/components/card/card.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="card" (click)="emitCardClickedEvent()">
```

Comment:
we are going to get smacked for all the divs... Generally we create sections and articles. This would have been a section because it's the cards and makes more semantic sense. But it's all good. Let's allow it this time. XD